data/curated/wa-cannabis/meta.yaml

```
@@ -0,0 +1,14 @@
+title: "Washington Cannabis Industry (2024–2025)"
+source: 
+  title: "Washington State Liquor and Cannabis Board (WSLCB) Reporting Data"
+  url: "https://lcb.wa.gov/"
+citation: 
+  title: "Ensign Analytics via yourweeddata.com"
+  url: "https://github.com/Ensign-Analytics"
+image: 
+  file: wa-cannabis-sample-plot.png
+  alt-text: "Bar chart showing cannabis retail sales in WA from 2024–2025"
+credit:
+  - name: Franklin Johnson
+    url: https://yourweeddata.com
+    type: curator
```

```suggestion
  post: [Franklin Johnson](https://github.com/Ensign-Analytics) via yourweeddata.com
  bluesky: https://bsky.app/profile/YOURNAME
  linkedin: https://www.linkedin.com/in/YOURNAME
  mastodon: @YOURNAME@YOURSERVER
  github: https://github.com/YOURNAME
```
The credit fields need to appear exactly as in the template. You can delete any that don't apply, though.